src/index.html

```diff
@@ -17,6 +17,40 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container-logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </div>
```

Comment:
The class name `container-logo` does not represent the meaning of the content. Class names should describe the content, not the styles or tag names. Consider renaming this class to something more descriptive.